DotBot.py

```diff
@@ -144,14 +143,13 @@ def _stopAndTransmit(self):
         self.currentSpeed        = 0
         
         # remember when I stop moving
-        self.tsMovementStop      = now
+        self.tsMovementStop      = self.simEngine.currentTime()
```

Comment:
why not using `now` here as well? You have a shorthand above to avoid having to call `self.simEngine.currentTime()` over and over, so why not use that throughout this method?